Sources/Plasma/PubUtilLib/plNetClientComm/plNetClientComm.cpp

```diff
@@ -1003,8 +1003,8 @@ void NetCommSetAccountUsernamePassword (
 
 //============================================================================
 void NetCommSetAuthTokenAndOS (
-    const char16_t*       authToken,
-    const char16_t*       os
+    char16_t              authToken[],
+    char16_t              os[]
```

Comment:
Why do the input parameters need to be non-`const`?

Reply:
I'm not actually sure why this was done. This change worked its way in as a part of a change for the intro movie which should be unrelated. Reverting.